### ISSUE TITLE
[css-paint-api] Computed vs. specified values in inputArguments

### ISSUE THREAD

**Issue by andruud** [w3c/css-houdini-drafts#896]
What is the rationale behind passing _specified_ values to the worklet via `inputArguments`? E.g. if I do `paint(foo, 1em)`, I'm currently supposed to get (if we ignore that it can't be reified, #895) a `CSSUnitValue` with `1em`. Isn't it more useful to have that `em` actually resolve against the font-size before passing it to the worklet?

cc @bfgeek

**Comment by tabatkins** [comment]
Yeah, custom functions (including these specialized ones we're providing) should be treated as computed values, and thus their argument should be computed as much as possible.
